sklearn/tests/test_isotonic.py

```diff
@@ -385,19 +385,19 @@ def test_isotonic_ymin_ymax():
                   -0.896, -0.377, -1.327, 0.180])
     y = isotonic_regression(x, y_min=0., y_max=0.1)
 
-    assert(np.all(y >= 0))
-    assert(np.all(y <= 0.1))
+    assert (np.all(y >= 0))
```

Comment:
if you're going to do a cosmetic clean-up here, it would be best to remove the parentheses.